### PR TITLE
merge from wix fork

### PR DESCRIPTION
**Pull request by alon-ne**
- Fix ICC optimization for webp
- Support webp nearlossless parameter
- Support SVG >=10MB
- Change Hue parameter to be float
- disable jpeg subsampling #pr
- shutdown vips properly
- add WebP orientation test

### CODE REVIEW

**Comment by tonimelisma** [comment]
Oh, a lot of good stuff here. I also see there's two other PRs. Let me know when they're ready to review/merge.

And thanks so much for the contributions! You rock! <3

**Comment by alon-ne** [comment]
If you're up for the task, I'd be glad if you could review this big PR (It also includes the webp ICC fix in the other PR).
Or, if you prefer, I and @Elad-Laufer can break it into a few smaller PR's. Other than being happy to contribute, once this is merged we won't have to maintain our own fork for Wix production code and that would make our lives easier.
So tell me which way would be easier for you. Thanks!

File: examples/jpeg/example_mozjpeg_image.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 	"io/ioutil"
 	"os"
 
-	"github.com/davidbyttow/govips/v2/vips"
```
Comment (tonimelisma) [inline examples/jpeg/example_mozjpeg_image.go:10]: I don't know how to keep the module paths separate. Can you use `replace` in go.mod? What can we do?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tonimelisma) [inline pom.xml:1]: Is this file relevant? Perhaps we can drop it from the PR

File: vips/color.go [hunk 3]
```diff
@@ -38,12 +37,25 @@ const (
 	InterpretationSRGB      Interpretation = C.VIPS_INTERPRETATION_sRGB
 	InterpretationYXY       Interpretation = C.VIPS_INTERPRETATION_YXY
 	InterpretationFourier   Interpretation = C.VIPS_INTERPRETATION_FOURIER
+	InterpretationGB16      Interpretation = C.VIPS_INTERPRETATION_RGB16
```
Comment (tonimelisma) [inline vips/color.go:40]: Should this be `InterpretationRGB16`

**Review by tonimelisma — CHANGES_REQUESTED** [review]
The changes look great. Thanks again a lot for the great contributions! Is it possible you summarise new functionality with a couple of bullet points so I can create a new release?

For InterpretationGB16 I know we need to be backwards compatible but if it's a typo maybe we can forget about it? Please see https://github.com/davidbyttow/govips/pull/181

It would be interesting to also get your thoughts on the recent developments and possible future architecture for govips. What are your thoughts on the loading parameters, animated GIFs, and the recent work on moving to `vips_object_set` which was done in PR #170?

Also, do you have any views on the discussion we had with David on issue #164? I suppose there's little value in refactoring the code.

**Comment by tonimelisma** [comment]
One final point: I think it's cool if we bump libvips and/or go version requirement, but can you also update the README correspondingly?

**Comment by Elad-Laufer** [comment]
> One final point: I think it's cool if we bump libvips and/or go version requirement, but can you also update the README correspondingly?

Yes it is

**Comment by davidbyttow** [comment]
Closing as per our conversation in email. Please do continue to send PRs from https://github.com/wix-incubator/govips-contrib and I will have a look!

**Pull request closed without merging**